src/otx/core/model/module/base.py

```diff
@@ -170,3 +175,19 @@ def meta_info(self) -> DataMetaInfo:
     @meta_info.setter
     def meta_info(self, meta_info: DataMetaInfo) -> None:
         self._meta_info = meta_info
+
+    def export(self, output_dir: str) ->  str:
```

Comment:
IMHO, it's better for `OTXModel` inherited class than `OTXLitModule`, such as `MMPretrainMulticlassClsModel`, to be in charge of export,
because `OTXLitModule` is a just wrapper class for torch-lightning trainer.
How do you think about it?

Reply:
we will have both export methods in LitModule and OTXModel. It is needed to not call model.model.export(), but just model.export()

Reply:
The initial idea is that `OTXLitModule` contains a reference export implementation, which can be customized down the hierarchy. Mmdeploy export would be implemented where it's required